docs/installation/prepare/rpi-setup.md

```diff
@@ -172,26 +172,31 @@ sudo apt upgrade
 ## Enable Communication Protocols
 
 If you are using hardware that requires I2C, SPI, serial, or one-wire protocols to communicate with your Pi, you will need to enable them using `raspi-config`.
+These protocols are required to support certain hardware, such as analog-to-digital converters (ADCs), accelerometers, and sensors.
 
-Launch the config tool by running the following command:
+1. Launch the config tool by running the following command:
 
-```sh {class="command-line" data-prompt="$"}
-sudo raspi-config
-```
+   ```sh {class="command-line" data-prompt="$"}
+   sudo raspi-config
+   ```
 
-Use your keyboard to select "Interface Options" and enable the relevant protocols.
+1. Use your keyboard to select "Interface Options", and press return.
 
-{{< imgproc alt="Screenshot of the Raspi Config screen with a red box and red arrow pointing to the '3 Interface Options' option where you can find the I2C and other drivers" src="/installation/rpi-setup/Installation-Raspberry-Pi-I2C-Raspi-Config-Interfacing-Options.png" resize="800x" declaredimensions=true >}}
+   {{< imgproc alt="Screenshot of the Raspi Config screen with a red box and red arrow pointing to the '3 Interface Options' option where you can find the I2C and other drivers" src="/installation/rpi-setup/Installation-Raspberry-Pi-I2C-Raspi-Config-Interfacing-Options.png" resize="800x" declaredimensions=true >}}
 
-{{< alert title="Important" color="note" >}}
-When using a CSI v1.3 or v2.0 camera, you need to enable legacy camera support.
-{{< /alert >}}
+1. Enable the relevant protocols to support your specific hardware:
 
-For these changes to take effect, you need to restart your Raspberry Pi if it hasn't already prompted you to do so.
+   - If you are using an analog-to-digital converter (ADC), enable **SPI**.
+   - If you are using an accelerometer, enable **I2C**.
```

Comment:
I feel this is too specific, LOTS of hardware use i2c.  I think we need to either give specific examples (and say "for example") or just tell them to check their hardware specs